vyper/builtins/interfaces/ERC4626.py

```diff
@@ -16,19 +16,24 @@
 # Functions
 @view
 @external
-def totalAssets() -> uint256:
+def asset() -> address:
     pass
 
 @view
 @external
-def convertToAssets(shareAmount: uint256) -> uint256:
+def totalAssets() -> uint256:
```

Comment:
Is this in the spec?

Reply:
I just changed the sequence - the diff is however displayed like that

![image](https://user-images.githubusercontent.com/25297591/220160391-faedb62d-281c-40df-966c-e7b313060e20.png)

Reply:
From the spec 
![image](https://user-images.githubusercontent.com/25297591/220160650-7c20a292-8a2d-4cb4-b738-e600761a11e5.png)